demos/python_demos/colorization_demo/colorization_demo.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+"""
+ Copyright (c) 2018 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from openvino.inference_engine import IENetwork, IECore
+import cv2 as cv
+import numpy as np
+import os
+from argparse import ArgumentParser, SUPPRESS
+import logging as log
+import sys
+
+
+def build_arg():
+    parser = ArgumentParser(add_help=False)
+    in_args = parser.add_argument_group('Options')
+    in_args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Help with the script.')
+    in_args.add_argument("-m", "--model", help="Required. Path to .xml file with pre-trained model.",
+                         required=True, type=str)
+    in_args.add_argument("-c", "--coeffs", help="Required. Path to .npy file with color coefficients.",
+                         required=True, type=str)
+    in_args.add_argument("-d", "--device",
+                         help="Optional. Specify target device for infer: CPU, GPU, FPGA, HDDL or MYRIAD. "
+                              "Default: CPU",
+                         default="CPU", type=str)
+    in_args.add_argument('-i', "--input",
+                         help='Required. Input to process.',
+                         required=True, type=str, metavar='"<path>"')
+    in_args.add_argument("-n", "--no_show", help="Optional. Disable display of results on screen.",
```

Comment:
`--no_show` doesn't need a short alias. None of the other demos have one.

Reply:
@IRDonch removed